### PR TITLE
Added classes for SCALED_IMU2 and SCALED_IMU3 to mavlink_messages.cpp.

### PR DESCRIPTION
**Pull request by mcsauder**
Hi

This PR restarts PR #9520 back to essentially the first commit of that PR.  It adds classes for the SCALED_IMU2 and SCALED_IMU3 messages to mavlink_messages.cpp and corrects a sensor_gyro_s variable type.

The hardware I am testing on only has two IMUs and I have not been able to test on hardware with 3 IMUs.  For convenience, below are screen captures from QGC mavlink inspector of the current upstream master and this PR showing the message output by adding SCALED_IMU, SCALED_IMU2, and SCALED_IMU3 to the config stream list of mavlink_main.cpp and connecting via USB to QGC.

The values reported back through SCALED_IMU and SCALED_IMU2 do not seem to be correct, believed to be a driver level bug yet to be resolved.  Up to the present I have been unable to create a working templated SCALED_IMU class implementation in mavlink_messages.cpp.  If successful in creating a templated solution to the additional messages I will submit a subsequent PR.

Please let me know if you have any questions on this PR.  Thanks!

-Mark

1)Current PX4/Firmware master branch:
 image

2) This PR
image

### CODE REVIEW

**Comment by LorenzMeier** [comment]
@mcsauder What is your testing status by now?

**Comment by mcsauder** [comment]
Hi @LorenzMeier , thanks for the ping!

This PR is functioning properly.  We've been flying this code for the past 10 days, about 5 flight hours on it just yesterday.  While this PR is complete, I am disappointed that I have been unable to get a templated version of this PR functioning.  Additional work remains elsewhere in the code base to deliver the proper values in these messages that this PR does not address.

Please let me know if you have additional questions about this PR!

-Mark

File: src/modules/mavlink/mavlink_messages.cpp [hunk 1]
```diff
@@ -2420,11 +2588,11 @@ class MavlinkStreamActuatorControlTarget : public MavlinkStream
 	uint64_t _att_ctrl_time;
 
 	/* do not allow top copying this class */
-	MavlinkStreamActuatorControlTarget(MavlinkStreamActuatorControlTarget &) = delete;
-	MavlinkStreamActuatorControlTarget &operator = (const MavlinkStreamActuatorControlTarget &) = delete;
+	MavlinkStreamActuatorControlTarget<N>(MavlinkStreamActuatorControlTarget<N> &) = delete;
```
Comment (LorenzMeier) [inline src/modules/mavlink/mavlink_messages.cpp]: Why is this change templated?

Comment (mcsauder) [reply src/modules/mavlink/mavlink_messages.cpp]: Agreed, apologies.  Fixed and pushed up.  Thanks for your review!

**Comment by mcsauder** [comment]
Hi @LorenzMeier, everything you identified has been addressed and pushed up. Thanks again for your feedback and let me know if you would like to see anything else addressed!

-Mark

**Comment by mcsauder** [comment]
@LorenzMeier , I'm just checking in on this PR.  Let me know if a better solution is required and if you can see one.  I've posted an issue in the mavlink repo that might lead to a better solution/implementation: https://github.com/mavlink/mavlink/issues/949

Let me know what you'd like to see, and thanks!

-Mark

**Comment by dagar** [comment]
This PR is fine given the current state of mavlink. I believe the main reason it wasn't merged the last time I looked is px4fmu-v2 flash constraints.

Looks like it's fitting at the moment.

**Pull request merged**